Misc mapping fixes and QoL additions

LovliestPlant opened the pull request:
## About The Pull Request

fix: #78135
fix: #78059

This PR remaps Birdshot's disposals room, makes several small fixes on Icebox and Metastation, adds cell timers to isolation cells where such cells are present (they don't open the door, effectively just an in-game timer) (in-cell flashes are now controlled with the timer, where applicable); and adds translator glove modules to the stacks of "accessibility" modules found in most security, medical, and engineering storage rooms. (adds these stacks in their entirety to Northstar).

Specific changes are as follows:
Birdshot
- Adds a roll of package paper to the cargo office
- Translator module [med,sec]
- Accessibility modules [eng]
- Recycler remap

Delta
- Translator module [med,sec,eng]
- Isolation cell timer

Icebox
- Translator module [med,sec,eng]
- Remove duplicate hand labeler on the rack near brig cells
- Adds a hand labeler to armory desk in gear room
- Isolation cell timer

Meta
- Translator module [med,sec,eng]
- Isolation cell timer
- Mends a broken corpse disposal pipe from aux surgery to the morgue

Northstar
- Accessibility modules [med,sec,eng]
- Nudges the binoculars off of the mass driver controls in ordnance
- Fixes the SM starting out hotwired (Rewires the SM room)

Tram
- Translator module [med,sec,eng]
- Isolation cell timers
## Why It's Good For The Game

Bug fixes with respect to Birdshot's recycler, Meta's corpse disposal, Northstar starting out hotwired, and Icebox's duplicated hand labeler.

Nudging Northstar's ordnance binoculars should make it a bit easier to find the mass driver controls.

Adding some packaging paper to Birdshot to make the techs' lives a little easier with a little less round-start fuss.

Adding a hand labeler to Icebox's gear room brings it in line with other maps in terms of rounds-start gear and locker labeling potential.

For players with characters running the Mute/Signer quirks, stock MODsuits are a pain to use. Suit gauntlets will replace their translator gloves. Unless they're able to get a suit put together ahead of time, they'll be stuck doing the retract gauntlets > send radio message > Extend Gauntlets shuffle. Adding a translator glove module to the stack of similar items (plasma fixation module / therma regulator) should alleviate that issue some.

Getting abandoned in an isolation cell sucks, and setting timers on your phone or some such is a hassle. Adding cell timers to isolation cells should go some way to alleviating those frustrations.
## Proof of Testing

<!-- Include any screenshots/videos/debugging steps of the code functioning successfully, between the </summary> and </details> code blocks. -->
<!-- To our mappers and spriters: Posting screenshots of content INSIDE EDITORS (aseprite, PDN, SDMM, ect) is NOT valid proof of testing. Please make sure that you COMPILE the game and provide PROOF you tested your edits. -->

<details>
<summary>Screenshots/Videos</summary>

### Birdshot

Disposal Room Remap
![bird_jani_v2](https://github.com/tgstation/tgstation/assets/107971606/aecc805f-08c9-469c-9963-860822c75f63)
Cargo Packing Paper
![tg-bird-packingPaper](https://github.com/tgstation/tgstation/assets/107971606/c0330acf-c64e-4dc4-9879-c7d8ae6047c4)
Engineering Accessibility Modules
![tg-bird-acc-eng](https://github.com/tgstation/tgstation/assets/107971606/ab055b28-2b40-453e-8850-1ceffb9c55ea)
Medbay Translator
![tg-bird-acc-med](https://github.com/tgstation/tgstation/assets/107971606/ecad5352-692d-4559-a1d3-4ee387fe449c)
Security Translator
![tg-bird-acc-sec](https://github.com/tgstation/tgstation/assets/107971606/045fa684-29f8-4112-ba58-59b90c135103)

### Deltastation

Engineering Translator
![tg-delta-acc-eng](https://github.com/tgstation/tgstation/assets/107971606/9289e303-e37a-4c11-b4c9-a6803cddcfd8)
Medbay Translator
![tg-delta-acc-med](https://github.com/tgstation/tgstation/assets/107971606/9a36819b-fbc4-4403-a0dd-199ba1c29cb3)
Security Translator
![tg-delta-acc-sec](https://github.com/tgstation/tgstation/assets/107971606/1d62d0d1-c564-4bfd-ad53-e41147682cba)
Isolation Cell Timer
![tg-delta-iso](https://github.com/tgstation/tgstation/assets/107971606/2c1579f4-d1a9-4d98-8e81-29b1cf0719d7)

### Icebox

Engineering Translator
![tg-ice-acc-eng](https://github.com/tgstation/tgstation/assets/107971606/9805b72e-cad6-4ddd-a7fd-adc271e6a341)
Medbay Translator
![tg-ice-acc-med](https://github.com/tgstation/tgstation/assets/107971606/8ab57572-0193-40c5-87ee-df95c7e5f9d8)
Security Translator
![tg-ice-acc-sec](https://github.com/tgstation/tgstation/assets/107971606/e234a98f-f429-4ed0-b465-3b795b1ff0bc)
Isolation Cell Timer
![tg-ice-iso](https://github.com/tgstation/tgstation/assets/107971606/9a0a7dc1-e369-46c8-8061-9c4635a63b5a)
Gear Room Hand Labeler
![tg-ice-label-armory](https://github.com/tgstation/tgstation/assets/107971606/36a58996-ac69-4978-8c79-eaa2478ce457)

### Metastation

Engineering Translator
![meta-acc-eng](https://github.com/tgstation/tgstation/assets/107971606/edbc746a-0c9c-4953-a744-1af064126c34)
Medbay Translator
![meta-acc-med](https://github.com/tgstation/tgstation/assets/107971606/a9b24f61-515e-40d1-b657-2a4b16920e51)
Security Translator
![meta-acc-sec](https://github.com/tgstation/tgstation/assets/107971606/55b91615-765e-42fe-adab-1a12e145ef48)
Isolation Cell Timer
![tg-meta-iso](https://github.com/tgstation/tgstation/assets/107971606/3bf6825c-0242-4332-ba71-db953a2e3902)

### Northstar

Engineering Accessibility Modules
![tg-north-acc-eng](https://github.com/tgstation/tgstation/assets/107971606/d32c1787-31e6-4ef7-964c-26eb87025888)
Medbay Accessibility Modules
![tg-north-acc-med](https://github.com/tgstation/tgstation/assets/107971606/fa3883f5-1e95-490a-b0b0-18ac08583221)
Security Accessibility Modules
![north-acc-sec](https://github.com/tgstation/tgstation/assets/107971606/d9308760-ac2f-4ae2-b91e-9d8dbcaaf0fd)
Supermatter Rewiring
![sm_annotate_2](https://github.com/tgstation/tgstation/assets/107971606/7c127678-6a55-454b-8e82-b615b41f0bcd)
Ordnance Binoculars
![tgqol_Northstar_Binos](https://github.com/tgstation/tgstation/assets/107971606/ce214728-48bf-436d-981e-bac40f8ca205)

### Tramstation

Engineering Translator
![tg-tram-acc-eng](https://github.com/tgstation/tgstation/assets/107971606/55b9993b-60b7-4e04-9073-0c8b3e7d9189)
Medbay Translator
![tg-tram-acc-med](https://github.com/tgstation/tgstation/assets/107971606/f4ac7a88-e3b1-4e4a-9914-70620c625b75)
Security Translator
![tg-tram-acc-sec](https://github.com/tgstation/tgstation/assets/107971606/8460cacb-a30a-45d0-b2bd-6c8666434055)
Isolation Cell Timer
![tg-tram-iso](https://github.com/tgstation/tgstation/assets/107971606/334be379-f6e6-45f0-93e9-b0e2f5d30b94)

</details>

## Changelog
:cl:
qol: [Deltastation, Icebox, Metastation, Tramstation] Adds cell timers to isolation cells. (they do not auto-open the doors)
qol: [Birdshot, Deltastation, Icebox, Metastation, Northstar, Tramstation] Adds translator glove modules to the stacks of "accessibility" (e.g. plasma fixation / thermal regulator) modules found in security, medical, and engineering storage rooms.
qol: [Birdshot] Adds a roll of packaging paper to the cargo office.
qol: [Icebox] Adds a hand labeler to security's gear room.
qol: [Northstar] Nudges the set of binoculars covering the mass driver controls in ordnance over a few inches.
fix: [Birdshot] Remaps the janitor's closet such that the recycling machine will now work.
fix: [Icebox] Removes a duplicated hand labeler from the rack near security's brig cells.
fix: [Metastation] Patches a broken corpse disposal pipe running from aux surgery to the morgue.
fix: [Northstar] Fixes the SM being hotwired at round-start (partially rewires the SM room, moves the APC to the North wall).
/:cl:

LovliestPlant committed:
fix batch 1

NS
- Ordnance binos
- SM hotwire (lower right maints)
Birdshot
- Recycler remap
Meta
- Broken corpse disposal pipe

LovliestPlant committed:
QoL batch

Birdshot
- Adds package paper to cargo
- Translator module [med,sec]
Delta
- Translator module [med,sec,eng]
- Isolation cell timer
Icebox
- Translator module [med,sec,eng]
- Remove duplicate hand labeler near brig cells
- Add hand labeler to armory desk in gear room
- Isolation cell timer
Meta
- Translator module [med,sec,eng]
- Isolation cell timer
Northstar
- Accessability modules [med,sec,eng]
Tram
- Translator module [med,sec,eng]
- Isolation cell timers

LovliestPlant committed:
review touchups

LovliestPlant committed:
second look touchups

- needless formatting change in Northstar
- adds engineering accessibility modules to birdshot

LovliestPlant committed:
snips one last bit of errant formatting

Sealed101 commented:
issues won't link because of incorrect formatting, use `fixes #foo` or similar

_For future reference, [these](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) are all the keywords that can be used to link issues/PRs._

LovliestPlant commented:
> issues won't link because of incorrect formatting, use `fixes #foo` or similar
> 
> _For future reference, [these](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) are all the keywords that can be used to link issues/PRs._

Good catch, thanks.

unit0016 commented:
Good catches all around, but one minor issue - This cuts off NorthStar's emitters from mains.

...Why isn't the NorthStar SM using multiple wiring layers anymore, anyways? I had used them explicitly to *not* have this happen, *and* to stop the pipes from masking the cabling.

LovliestPlant commented:
> Good catches all around, but one minor issue - This cuts off NorthStar's emitters from mains, and has the SM's own output play the role, which is.. a catch 22 without a pacman.
> 
> ...Why isn't the NorthStar SM using multiple wiring layers anymore, anyways? I had used them explicitly to _not_ have this happen, _and_ to stop the pipes from masking the cabling.

Taking another look, the emitters should be fine. The red section bridging into maints is what was removed.

![sm_annotate](https://github.com/tgstation/tgstation/assets/107971606/991b705a-1e99-4b1c-b7eb-1b9bc7ceeee8)

rdragan commented:
> > Good catches all around, but one minor issue - This cuts off NorthStar's emitters from mains, and has the SM's own output play the role, which is.. a catch 22 without a pacman.
> > ...Why isn't the NorthStar SM using multiple wiring layers anymore, anyways? I had used them explicitly to _not_ have this happen, _and_ to stop the pipes from masking the cabling.
> 
> Taking another look, the emitters should be fine. The red section bridging into maints is what was removed.
> 
> ![sm_annotate](https://user-images.githubusercontent.com/107971606/266474734-991b705a-1e99-4b1c-b7eb-1b9bc7ceeee8.PNG)

The problem is that the emitters are connected to the tesla coils instead of the main power grid. Emitters rely on getting power from the roundstart SMES charge to jumpstart the SM, with you modifications they will be unpowered. Basically you want to have a separation like in my screenshot but this doesnt look like it's doable without moving some walls, having the cable running under rwalls or using a separate cable layer.

![image](https://github.com/tgstation/tgstation/assets/43266738/251d927a-cf9f-46f9-b7dc-19527628fb3d)

LovliestPlant committed:
rewires SM room

- Routes emitter mains connection in through southern maints access
- Moves the room APC to the north wall, hooks into mains leading out into north mains
- Re-routes tesla cables along the left side of the SM chamber

LovliestPlant committed:
linters fix

LovliestPlant commented:
> > > Good catches all around, but one minor issue - This cuts off NorthStar's emitters from mains, and has the SM's own output play the role, which is.. a catch 22 without a pacman.
> > > ...Why isn't the NorthStar SM using multiple wiring layers anymore, anyways? I had used them explicitly to _not_ have this happen, _and_ to stop the pipes from masking the cabling.
> > 
> > 
> > Taking another look, the emitters should be fine. The red section bridging into maints is what was removed.
> > ![sm_annotate](https://user-images.githubusercontent.com/107971606/266474734-991b705a-1e99-4b1c-b7eb-1b9bc7ceeee8.PNG)
> 
> The problem is that the emitters are connected to the tesla coils instead of the main power grid. Emitters rely on getting power from the roundstart SMES charge to jumpstart the SM, with you modifications they will be unpowered. Basically you want to have a separation like in my screenshot but this doesnt look like it's doable without moving some walls, having the cable running under rwalls or using a separate cable layer.
> 
> ![image](https://user-images.githubusercontent.com/43266738/266557919-251d927a-cf9f-46f9-b7dc-19527628fb3d.png)

oh right, that makes sense. So with a little bit of reshuffling, I think this should have everyone happy and powered without undue fuss. Basic changes are:
- Routes emitter mains connection in through southern maints access
- Moves the room APC to the north wall, hooks into mains leading out into north mains
- Re-routes tesla cables along the left side of the SM chamber

![sm_annotate_2](https://github.com/tgstation/tgstation/assets/107971606/f2fd8140-739f-4fda-a0be-b2e88a3e23b8)

LovliestPlant committed:
Update tramstation.dmm

Maurukas reviewed (changes requested):
One minor issue, otherwise looks good.

<details>
<summary>
Birdshot
</summary>

Please shift this door in the custodial closet a few tiles to the left. It is blocked off by the stacker and chute in this position.

![image](https://github.com/tgstation/tgstation/assets/66576896/857df01a-4c9a-42df-a866-3cc30251b771)
</details>

LovliestPlant committed:
Tweaks Birdshot's Recylcing room

LovliestPlant commented:
room reshuffled a bit to make everything fit.
![bird_jani_v2](https://github.com/tgstation/tgstation/assets/107971606/4307545e-d00a-497d-8d4a-10752ae57842)
